docs/current_docs/api/remote-modules.mdx

```diff
@@ -66,6 +66,96 @@ If this works, Dagger will be able to use the same credentials to access your pr
 - Azure DevOps: Use [Git Credential Manager](https://learn.microsoft.com/en-us/azure/devops/repos/git/set-up-credential-managers)
 - BitBucket: Configure credentials using the [Git credential system](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage) (the widely-adopted implementation is [Git Credential Manager](https://github.com/git-ecosystem/git-credential-manager))
 
+### Using credentials in CI environments
+
+When running Dagger inside CI systems, you often have tokens provided by the CI provider. Instead of writing a custom helper script, you can directly store credentials in an ephemeral credentials file and configure Git to use it. Below are examples for GitLab, GitHub, and BitBucket, each shown in separate tabs. These approaches are minimal and avoid persistent storage of credentials.
```

Comment:
We also mention Azure DevOps in a previous section so we should add a tab for that provider as well here for consistency.